episodes/superspreading-estimate.Rmd

```diff
@@ -0,0 +1,836 @@
+
+- When the 20% most infectious cases contribute to the 80% of transmission (or more), there is a high individual-level variation in transmission, with a highly overdispersed offspring distribution ($k<0.3$), e.g., SARS-1.
+
+- When the 20% most infectious cases contribute to the ~50% of transmission, there is a low individual-level variation in transmission, with a moderately dispersed offspring distribution ($k > 3$), e.g. Ebola.
+
+```{r,message=FALSE,warning=FALSE,echo=FALSE}
+library(epiparameter)
+library(tidyverse)
+
+epidist_string <- epidist_db(
+  epi_dist = "offspring distribution"
+) %>%
+  list_distributions() %>%
+  dplyr::select(disease) %>%
+  distinct()
+
+across_offspring <- epidist_string %>%
+  mutate(
+    epidist_out =
+      map(
+        .x = disease,
+        .f = epiparameter::epidist_db,
+        epi_dist = "offspring distribution",
+        single_epidist = TRUE
+      )
+  ) %>%
+  mutate(
+    epidist_params =
+      map(
+        .x = epidist_out,
+        .f = epiparameter::get_parameters
+      )
+  ) %>%
+  unnest_wider(col = epidist_params) %>%
+  expand_grid(percent_transmission = seq(0.01, 1, 0.01)) %>%
+  # mutate(percent_transmission = 0.8) %>%
+  mutate(
+    transmission_output =
+      pmap(
+        .l = select(., R = mean, k = dispersion, percent_transmission),
+        .f = superspreading::proportion_transmission,
+        format_prop = FALSE
+      )
+  ) %>%
+  unnest_wider(col = transmission_output) %>%
+  rowwise() %>%
+  mutate(
+    percent_cases =
+      sum(
+        c_across(cols = starts_with("prop_")),
+        na.rm = TRUE
+      )
+  ) %>%
+  select(-starts_with("prop_"))
+
+across_offspring_tip <- across_offspring %>%
+  group_by(disease) %>%
+  filter(percent_transmission < 0.98, percent_transmission > 0.85) %>%
+  slice_max(percent_transmission) %>%
+  ungroup() %>%
+  mutate(disease = case_when(
+    str_detect(disease, stringr::fixed("Hantavirus")) ~ "Hantavirus",
+    str_detect(disease, stringr::fixed("Ebola")) ~ "Ebola",
+    TRUE ~ disease
+  ))
+
+across_offspring %>%
+  ggplot() +
+  geom_line(
+    aes(
+      x = percent_cases,
+      y = percent_transmission,
+      color = dispersion,
+      group = disease
+    )
+  ) +
+  geom_text(
+    data = across_offspring_tip,
+    aes(
+      x = percent_cases,
+      y = percent_transmission,
+      label = disease
+    ),
+    hjust = 0.0,
+    vjust = 1.0,
+    angle = 25,
+    size = 3
+  ) +
+  scale_y_continuous(breaks = scales::breaks_pretty(n = 5)) +
+  colorspace::scale_color_continuous_diverging(trans = "log10") +
+  labs(
+    x = "Proportion of infectious cases (ranked)",
+    y = "Expected proportion of transmission",
+    color = "Dispersion\nparameter (k)"
+  ) +
+  # geom_hline(aes(yintercept = 0.8),lty = 3) +
+  geom_vline(aes(xintercept = 0.2), lty = 2) +
+  coord_fixed(ratio = 1)
+```
+
+
+::::::::::::::::::::::::::::::::::
+
+## Backward and forward contact tracing
```

Comment:
This is quite sudden. You could retitle it as "Controlling superspreading events with contact tracing".

Reply:
done in https://github.com/epiverse-trace/tutorials-middle/commit/2679788c061c063b3f26fbfffad770621fe57f6c